packages/authorization-updater/src/index.ts

```diff
@@ -269,7 +269,6 @@ export async function sendPurposeAuthUpdate(
           "PurposeVersionUnsuspendedByProducer",
           "PurposeVersionOverQuotaUnsuspended",
           "NewPurposeVersionActivated",
-          "NewPurposeVersionWaitingForApproval",
           "PurposeVersionRejected",
           "PurposeVersionActivated",
           "PurposeArchived"
```

Comment:
I think we should also move the `PurposeVersionRejected` message to the no-ops, since I could have a rejected version while continuing having an active version that continues  delivering tokens

Reply:
I agree.

Probably also `PurposeWaitingForApproval` is a no-op? 🤔 
It is triggered from -> Draft to WaitingForApproval, so no changes are required (especially now that Drafts cannot be assigned to clients)

Reply:
Makes sense!